src/widgetastic/browser.py

```diff
@@ -440,7 +440,7 @@ def set_attribute(self, attr, value, *args, **kwargs):
             "arguments[0].setAttribute(arguments[1], arguments[2]);",
             self.element(*args, **kwargs), attr, value)
 
-    def size(self, *args, **kwargs):
+    def size_of(self, *args, **kwargs):
```

Comment:
additional note, might it make sense to use a named-tuple as a result here? 
the usage points a few lines down are seemingly hard to read due to magic number